include/autocluster.hrl

```diff
@@ -20,6 +20,9 @@
          {config, longname,              "RABBITMQ_USE_LONGNAME",  false,        atom,    false},
          {config, node_name,             "RABBITMQ_NODENAME",      "rabbit",     string,  false},
          {config, node_type,             "RABBITMQ_NODE_TYPE",     disc,         atom,    false},
+         {config, http_proxy,            "HTTP_PROXY",            "undefined",   string,  false},
+         {config, https_proxy,           "HTTPS_PROXY",           "undefined",   string,  false},
+         {config, no_proxy,              "NO_PROXY",              "undefined",   string,  false},
```

Comment:
The no_proxy was taken from Erlang `httpc` APIs. I will change it in `proxy_exclusion_list`.